terraform/geomatch_app/alb/main.tf

```diff
@@ -61,6 +47,20 @@ resource "aws_vpc_security_group_ingress_rule" "inbound_443" {
   }
 }
 
+resource "aws_vpc_security_group_egress_rule" "outbound_lambda" {
```

Comment:
What do you think of the name "internal" instead of "outbound_lambda"? Really, this SG could be used for any service that needs to connect to the ALB, not just a lambda